lib/test.js

```diff
@@ -240,11 +247,11 @@ Test.prototype._publicApi = function () {
 	}
 
 	// Patched assert methods: increase assert count and store errors.
-	Object.keys(assert).forEach(function (el) {
+	Object.keys(Assert.prototype).forEach(function (el) {
 		api.skip[el] = skipFn;
 		api[el] = function () {
 			try {
-				var fn = assert[el].apply(assert, arguments);
+				var fn = self.assert[el].apply(self.assert, arguments);
```

Comment:
Do we have to make `assert` a class. Can't we just track assertions here? Demanding the assert class track assertions is going to make it harder to support custom assertions down the road.

Is the issue simply that it is hard to know which argument is the message here?